components/blocks/Blocks.tsx

```diff
@@ -79,6 +83,14 @@ export const Blocks = ({
         return <TestimonialsBlock key={`block-${index}`} data={block} index={index} />;
       case 'PageBlocksQuote':
         return <QuoteBlock key={`block-${index}`} data={block} index={index} />;
+      case 'PageBlocksTextMediaColumnComponent':
+        return <TextAndMediaColumnsComponent key={`block-${index}`} data={block} />;
+      case 'PageBlocksImageRow':
+        return <ImageRowComponent key={`block-${index}`} data={block} />;
+      case 'PageBlocksIconBanner':
+        return <IconBannerComponent key={`block-${index}`} data={block} />;
+      case 'PageBlocksSpacer':
+        return <SpacerComponent key={`block-${index}`} data={block} />;
```

Comment:
Make a PBI to investigate these hardcoded strings - I'm confident there is a better approach